package/batocera/core/batocera-configgen/configgen/configgen/generators/libretro/libretroConfig.py

```diff
@@ -353,6 +373,12 @@ def createLibretroConfig(system, controllers, rom, bezel, gameResolution):
             retroarchConfig['netplay_password'] = '"' + systemConfig.get("netplay.password", "") + '"'
             retroarchConfig['netplay_spectate_password'] = '"' + systemConfig.get("netplay.spectatepassword", "") + '"'
 
+        # Netplay hide the gameplay
+        if system.isOptSet('netplay.public') and system.getOptBoolean('netplay.public') == False:
```

Comment:
Please remove point in the variable name & replace it with _ ( or just rename to netplay_public_announce )

Reply:
Done